src/test/java/org/candlepin/model/ConsumerCloudDataTest.java

```diff
@@ -12,63 +12,59 @@
-import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertSame;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
 import org.candlepin.test.TestUtil;
 
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
 
 import java.util.Arrays;
 import java.util.Collection;
-import java.util.Collections;
 import java.util.List;
 
+
+
 public class ConsumerCloudDataTest {
 
     @Test
-    public void testSetId() {
+    public void testSetCloudProviderShortName() {
         ConsumerCloudData consumerCloudData = new ConsumerCloudData();
-        String validId = "12345";
-        consumerCloudData.setId(validId);
-        assertEquals(validId, consumerCloudData.getId());
+        String validName = "aws";
+        consumerCloudData.setCloudProviderShortName(validName);
+        assertEquals(validName, consumerCloudData.getCloudProviderShortName());
+    }
 
-        Exception exception = assertThrows(IllegalArgumentException.class, () -> {
-            consumerCloudData.setId(null);
-        });
-        assertEquals("ID is null or blank", exception.getMessage());
+    @ParameterizedTest
+    @NullAndEmptySource
+    @ValueSource(strings = { " ", "   ", "\t" })
+    public void testSetCloudProviderShortNameWithNullAndEmptyInputs(String value) {
+        ConsumerCloudData ccdata = new ConsumerCloudData();
 
-        exception = assertThrows(IllegalArgumentException.class, () -> {
-            consumerCloudData.setId("");
+        Exception exception = assertThrows(IllegalArgumentException.class, () -> {
+            ccdata.setCloudProviderShortName(value);
         });
-        assertEquals("ID is null or blank", exception.getMessage());
 
-        exception = assertThrows(IllegalArgumentException.class, () -> {
-            consumerCloudData.setId("123456789012345678901234567890123");
-        });
-        assertEquals("ID exceeds the max length", exception.getMessage());
+        assertEquals("cloudProviderShortName is null or empty", exception.getMessage());
     }
 
     @Test
-    public void testSetCloudProviderShortName() {
-        ConsumerCloudData consumerCloudData = new ConsumerCloudData();
-        String validName = "aws";
-        consumerCloudData.setCloudProviderShortName(validName);
-        assertEquals(validName, consumerCloudData.getCloudProviderShortName());
+    public void testSetCloudProviderShortNameWithLongValue() {
+        ConsumerCloudData ccdata = new ConsumerCloudData();
+
+        String value = "a".repeat(ConsumerCloudData.CLOUD_PROVIDER_MAX_LENGTH + 1);
 
         Exception exception = assertThrows(IllegalArgumentException.class, () -> {
-            consumerCloudData.setCloudProviderShortName(null);
+            ccdata.setCloudProviderShortName("1234567890123456");
```

Comment:
Use the "value" variable here instead of the "1234567890123456" string.

Reply:
Argh! I even re-wrote the whole test to avoid this very mistake. Fixed.

Reply:
I figured that you probably refactored the test haha